3.7/release-notes-known-issues36.md

```diff
@@ -54,7 +55,7 @@ Other
-| **Date Added:** 2019-12-10 <br> **Component:** HTTP API <br> **Deployment Mode:** Cluster <br> **Description:** The endpoint `/_admin/cluster/rebalanceShards` does not allow to rebalance shards of a database other than `_system`. This issue was fixed for 3.5, but not ported to 3.6.0-rc.1. <br> **Affected Versions:** 3.6.0-rc.1 <br> **Fixed in Versions:** 3.5.3, 3.6.0-rc.2 <br> **Reference:** [arangodb/arangodb#10668](https://github.com/arangodb/arangodb/pull/10668){:target="_blank"} |
-| **Date Added:** 2019-12-10 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The DMG package for macOS is unnotarized, which prevents the execution of _ArangoDB3-CLI.app_ under macOS 10.15 (Catalina) with error message: "_ArangoDB3-CLI_ can't be opened because Apple cannot check it for malicious software" <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.0-rc.1 <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10561](https://github.com/arangodb/arangodb/issues/10561){:target="_blank"} |
-| **Date Added:** 2019-12-10 <br> **Component:** OneShard <br> **Deployment Mode:** Cluster <br> **Description:** Collections can be created in OneShard databases with a replication factor higher than the allowed maximum (number of currently present DB-Servers), effectively ignoring the `enforceReplicationFactor` option. <br> **Affected Versions:** 3.6.0-rc.1 <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10655](https://github.com/arangodb/arangodb/pull/10655){:target="_blank"} |
+| **Date Added:** 2019-12-10 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** The DMG package for macOS is not notarized, which prevents the execution of _ArangoDB3-CLI.app_ under macOS 10.15 (Catalina) with error message: "_ArangoDB3-CLI_ can't be opened because Apple cannot check it for malicious software" <br> **Affected Versions:** 3.3.x, 3.4.x, 3.5.x, 3.6.x <br> **Fixed in Versions:** - <br> **Reference:** [arangodb/arangodb#10561](https://github.com/arangodb/arangodb/issues/10561){:target="_blank"} |
+| **Date Added:** 2020-01-07 <br> **Component:** Installer <br> **Deployment Mode:** All <br> **Description:** V8-based binaries of the client packages (arangosh, arangoinspect, foxx-manager) have an incorrect default value for `javascript.startup-directory` and will thus not find the required JavaScript folder unless specified by the user. <br> **Affected Versions:** 3.4.x, 3.5.x, 3.6.0 <br> **Fixed in Versions:** 3.4.10, 3.5.5, 3.6.2 <br> **Reference:** [arangodb/release-qa#183](https://github.com/arangodb/release-qa/issues/183){:target="_blank"} (internal) |
```

Comment:
I think it should be 3.4.9, 3.5.4 and 3.6.1 here?

Reply:
3.4.10, 3.5.5, 3.6.1 as per Vadim (changes too close to 3.4/3.5 release, will not be included)